README.md

```diff
@@ -55,57 +53,79 @@ Here are the examples,
 ```js
 For all blog list,
 
-  <BlogList :showDrafts="true" />
+  <BlogList
+      :showDrafts="true"
+      :posts="posts"
+      :featurePosts="featurePosts"
+    />
 
 For tag list,
-  <TagList :slug="slug" @notfound="<your-callback>" :showDrafts="true"  />
+  <TagList :slug="slug"
+      @notfound="<your-callback>"
+      :showDrafts="true"
+      :tagName:"tagname" />
 
 
 For Blog details,
   <BlogDetail
       :slug="slug"
       @notfound="<your-callback>"
-      :showDrafts="true"
-      :iframely-key="IFRAMELY_KEY"
+      :post="post"
       :recaptcha-key="RECAPTCHA_SITE_KEY"
+      :status="status"
+
+
+      :recommandedPosts="recommandedPosts"
+      :websiteUrl="websiteUrl"
     />
 
 ```
+
+Here `websiteUrl` is the link of the user website to share the links of blog on different platforms.
+
 <br>
 
-# Steps to Run the component 
+# Steps to Run the component
 
 ## Install dependencies
 
-
 ```bash
-yarn install 
+yarn install
 ```
+
 #### To initialize the Nuxt repo :
+
 ```bash
 npx nuxi prepare
 ```
+
 ## To create output.css file
-``` bash
+
+```bash
 yarn prepack
 ```
+
 - Copy `output.css` file from `.dist/` folder and paste inside the `src` folder
+
 ## Start component in dev mode using :
 
 ```bash
-yarn dev 
+yarn dev
 ```
+
 ## Compiles and minifies for production :
 
 ```bash
-yarn dev:build 
+yarn dev:build
 ```
+
 ```bash
 yarn generate
```

Comment:
Let's remove this, as this will not work for this project and add `yarn dev:prepare` at suitable place